MediaRecorder.html

```diff
@@ -462,8 +462,9 @@
              </a> [[!HTML5]]. The codecs parameter [[!RFC6381]], if necessary to
              specify exactly how the resource is encoded, SHOULD be added
              to <code>type</code>. If <var>type</var> can be expressed in several
-             equivalent ways (e.g. with or without quotes), the shortest form
-             and lowercase letters SHOULD be preferred.
+             equivalent ways (e.g. with or without quotes or whitespaces), the
+             one using the smallest amount of lowercase characters MUST be
```

Comment:
Heh I have written all this so many times that I just don't know what I'm typing. I rewrote again this section but, if you still have comments (all welcome), may I ask you suggest an example wording? ;)